jansi/src/main/java/org/fusesource/jansi/AnsiConsole.java

```diff
@@ -129,7 +129,7 @@ public void close() throws IOException {
         };
     }
 
-    private static boolean isCygwin() {
+    private static boolean isXterm() {
```

Comment:
Yes, it's unrelated to the global topic of the PR, but then again that's why it's a separate commit. As it's such a small change I did not bother to explicitly mention it in the PR's description.